bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/ThingFactory.java

```diff
@@ -193,6 +194,9 @@ private static Channel createChannel(ChannelDefinition channelDefinition, ThingU
             }
         }
 
+        // TODO: Is this safe, or should I make a copy?
+        channelBuilder = channelBuilder.withProperties(channelDefinition.getProperties());
```

Comment:
Good question. I think a copy would be definitely safer. But I would expect that the channel builder creates a copy.